### ISSUE TITLE
Material concentration

### ISSUE THREAD

**Issue by lunar-crater-ex** [citadel-station/sector-xiv#7]
# WAITING ON WORLDGEN MERGE.
## Initial requirements
tl;dr ore isn't uniform.

This involves using noise channels to control the concentration of different ore types within asteroid rock. Use NoiseIndexSystem and a noise channel per "group" of ore to essentially function as a multiplier for the probablity of that ore dropping. I personally would use a `Dictionary<string, EntitySpawnEntry>` for this, storing the rock's initial world position upon spawn and then computing the `List<EntitySpawnEntry>` to use by going over the keys/values of the dictionary, checking the value of the noise channel (the key) at that initial location, and multiplying the spawn entry's probability by that value.

## Future work
Bonus points if you make asteroid debris decide their IFF color based on which ore noise channel has the largest value at the center of the roid. (And even more bonus points if, instead, you implement a scanning system of sorts to locate concentrations of specific ores without simply being a map)
